app/src/lib/editors/win32.ts

```diff
@@ -357,7 +357,7 @@ const editors: WindowsExternalEditor[] = [
     registryKeys: [Wow64LocalMachineUninstallKey('RStudio')],
     installLocationRegistryKey: 'DisplayIcon',
     displayNamePrefix: 'RStudio',
-    publisher: 'RStudio',
+    publisher: 'Posit Software',
```

Comment:
After @voidei 's work in #15585 we can just have multiple publishers, which should solve this problem! 🎉  😄 

```suggestion
    publishers: ['RStudio', 'Posit Software'],
```